src/lib.rs

```diff
@@ -490,3 +494,54 @@ fn generate_body(trait_: &DeriveTrait, item: &Item) -> TokenStream {
 		}
 	}
 }
+
+/// Removes the `derive_where` attributes from all fields, and variants.
+///
+/// This is needed, because rust does not support those for proc_macro_attribute
+/// currently
```

Comment:
```suggestion
/// Removes `derive_where` attributes from the item and all fields and variants.
///
/// This is necessary because Rust currently does not support helper attributes for attribute proc-macros and therefore doesn't automatically remove them.
```